joern-cli/frontends/swiftsrc2cpg/src/main/scala/io/joern/swiftsrc2cpg/astcreation/AstForExprSyntaxCreator.scala

```diff
@@ -50,8 +50,14 @@ trait AstForExprSyntaxCreator(implicit withSchemaValidation: ValidationMode) {
     callAst(callNode_, argAsts)
   }
 
-  private def astForAssignmentExprSyntax(node: AssignmentExprSyntax): Ast         = notHandledYet(node)
-  private def astForAwaitExprSyntax(node: AwaitExprSyntax): Ast                   = notHandledYet(node)
+  private def astForAssignmentExprSyntax(node: AssignmentExprSyntax): Ast = notHandledYet(node)
+
+  private def astForAwaitExprSyntax(node: AwaitExprSyntax): Ast = {
+    val callNode_ = callNode(node, code(node), "<operator>.await", DispatchTypes.STATIC_DISPATCH)
```

Comment:
Only feedback is that this operator could perhaps be somewhere in a `Defines` class or something

Reply:
Right! I will do a major refactoring for swiftsrc2cpg once it handles most of the language constructs my local testing with large Swift apps reveals (currently testing all the stuff on https://github.com/apple/swift itself).